.github/workflows/main-deploy-docs.yml

```diff
@@ -21,5 +21,6 @@ jobs:
       - name: Docs
         env:
           DOCS_TOKEN: ${{ secrets.DOCS_TOKEN }}
+          TIBANNA_VERSION: 2.2.0
```

Comment:
The docs-deploy GA does not work anymore. I think that's because it can't determine the version, since it is now parsed from pyproject.toml. One solution would be to set the env variable here. Not elegant at all, but I am not sure how to get it otherwise.
I had the same issue with the Lambda function and solved it there via env variables.